runtime/j9vm/jvm.c

```diff
@@ -1469,18 +1471,14 @@ getVersionFromReleaseFile(void)
 	if (NULL != propsFile) {
 		const char *version = props_file_get(propsFile, "JAVA_VERSION");
```

Comment:
Not sure any of this is useful, we build specifically for the openjdk version. How about deleting this code to make future ports that must easier.

Reply:
Yeah, it can be simpler though we need consider the impact on legacy non-openj9 Java 8 builds.

Reply:
I thought that uses classlib.properties?

Reply:
Seems getVersionFromReleaseFile() and the caller can be deleted and it would be fine.